streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -92,7 +93,7 @@
 
     /**
      * Materialize this stream to a topic, also creates a new instance of {@link KStream} from the topic
-     * using default serializers and deserializers.
+     * using default serializers and deserializers and hash partitioning on the key.
```

Comment:
`DefaultPartitioner` seems not to be correct here. Interface `StreamPartitioner` is implemented by `WindowedStreamPartitioner`. It seems that `WindowStreamPartitioner` is expected to behave the same way a `DefaultPartitioner` but it seems strange to claim `DefaultPartitioner` is use, if the class does not implement `StreamPartitioner`. What do you think?